demisto_sdk/commands/init/initiator.py

```diff
@@ -95,6 +96,13 @@ def __init__(self, output: str, name: str = '', id: str = '', integration: bool
 
     HELLO_WORLD_INTEGRATION = 'HelloWorld'
     HELLO_WORLD_SCRIPT = 'HelloWorldScript'
+    HELLO_WORLD_SCRIPT_FILES = ['HelloWorldScript.py', 'HelloWorldScript.yml', 'HelloWorldScript_test.py']
+    HELLO_WORLD_INTEGRATION_FILES = ['HelloWorld.py', 'HelloWorld.yml', 'HelloWorld_description.md',
+                                     'HelloWorld_image.png', 'HelloWorld_test.py', 'Pipfile', 'Pipfile.lock',
+                                     'test_data/domain_reputation.json', 'test_data/get_alert.json',
+                                     'test_data/ip_reputation.json', 'test_data/scan_results.json',
+                                     'test_data/search_alerts.json', 'test_data/update_alert_status.json']
```

Comment:
use `os.path.join` for all the files under `test_data`. not sure if this will work on windows machines (you can ask someone from the security team for help testing it).